.npmrc

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}
```

Comment:
Whats the purpose of this file? I think it should be generated by CI if its for publishing. Otherwise it will conflict with local npm settings

Reply:
Not publishing; required to install @ vercel/style-guide since its a private module

Reply:
Lets try to get it public. I commented on another issue to see what Brody thinks.

Reply:
Yes, the intention was to publish this as non-private. I'll get that done tomorrow.

Reply:
Bumping this - i don't think its publicly available yet